docs/Administration.rst

```diff
@@ -2456,18 +2460,45 @@ should be sent to you.
 Administrating spaces 
 ======================
 
-It is possible to restrict which users can create and/or manage spaces
+It is possible to restrict which users can create and/or manage spaces and its templates
 through an easy to use interface .
 
 |image305|
 
-.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and only administrators (i.e.
-          the members of the group ``*:/platform/administrators``) are allowed to manage spaces.
+.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and templates and only administrators (i.e.
+          the members of the group ``*:/platform/administrators``) are allowed to manage spaces and its templates.
           
 The spaces administration interface is accessible via the menu Administration --> Spaces.
 
 |image306|
 
+This Spaces menu contains two choices:
+
+- "Manage Templates" : The administrator will have the ability to create, update, delete or even change permissions of some space templates.
+- "Manage Spaces" : The administrator tab displays the list of groups who can :ref:`Create <Create-Space>` or :ref:`Manage <AdministratingSpaces.ManageSpaces>` spaces. 
+
+Manage Templates
+~~~~~~~~~~~~~~~~~~
+|image322|
+
+The "Manage Templates" interface will display all templates that can be chosen by all space creators.
+
+All templates will be presented in a table with: 
+- Template Name
+- Template Description
+- Hidden status (Read Only switch button)
+- Registration
+- Applications
+- Banner link (specific banner for the space template)
+- Permissions (permission to view the template in the templates list in the space creation popup)
+
+.. note:: The Manage Template table is only for displaying all templates informations, to edit the template propreties, you have to do it manually.
```

Comment:
"manually" is not clear here. If you mean by configuration, you should link to the documentation that explains how to customize tempaltes.

Reply:
I've just add it and wait for its documentation in the developer guide so that i can link it (not ready yet).